test/performance/odp_sched_pktio_run.sh

```diff
@@ -81,10 +82,15 @@ run_sched_pktio()
 		echo "PASS: received and transmitted over 5000 packets"
 		ret=0
 	else
-		echo "FAIL: less than thousand rx or tx packets $ret"
+		echo -e "\nodp_packet_gen"
+		echo "=============="
+		cat $GEN_LOG
+		echo "FAIL: less than 5000 rx or tx packets: ret=$ret"
```

Comment:
Maybe the explicit "5000" could be swapped to something like a "threshold" to avoid a slightly fragile dependency to the `odp_sched_pktio.c` constant.

Reply:
Updated in V2.